apps/packages/package_service.py

```diff
@@ -159,13 +159,14 @@ def check_not_in_personal_space(self, docs):
         items if it can't remove it from the packages it belongs to.
         """
         for doc in docs:
-            if not doc.get('task') or not doc['task'].get('desk'):
-                message = 'Packages can not be created in the personal space.'
-                logger.error(message)
-                raise SuperdeskApiError.forbiddenError(message=message)
-            if not doc['task'].get('stage'):
-                desk = get_resource_service('desks').find_one(req=None, _id=doc['task']['desk'])
-                doc['task']['stage'] = desk['working_stage']
+            if not doc.get('particular_type'):
```

Comment:
:+1:  I'll add a comment similar to the following
`particular_type` is a liveblog specific field telling us weather the material that we are posting is "item"  or "post". For us, a post is  a package and consists in one or more items.
Since we don't have tasks and desks in liveblog, posting in the personal space verification is not needed so this `particular_type` check-up is preventing the personal space verification to be executed.